content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+---
+I﻿n this new article that is part of this series dedicated to the [open sourcing of our Workshops-on-Demand project](https://developer.hpe.com/blog/willing-to-build-up-your-own-workshops-on-demand-infrastructure/), I will focus on the necessary steps to build up a new workshop. I already covered most of the infrastructure part that supports the workshops. It now makes sense to emphasize a bit on the content creation.
+
+# O﻿verview
+
+H﻿ere is a simple flowchart describing the 10000 feet view of the creation process:
+
+![](/img/wod-b-process.png "Workshop's creation flow.")
+
+A﻿s you can see, no rocket science here. Just common sense. Depending on the workshop you wish to create, some obvious requirements should show up. A workshop based on a programmatic language for instance, will require the relevant kernel to be setup on the JupyterHub server. The following [page](https://gist.github.com/chronitis/682c4e0d9f663e85e3d87e97cd7d1624) will list all available kernels.
+
+S﻿ome other workshops might need a proper infrastructure to run on. A kubernetes101 workshop fro instance could not exist without the presence of a proper Kubernetes cluster. Same thing goes for any HPE related solutions.
+
+F﻿rom an infrastructure standpoint, a minimum of environments are necessary. You will need a test/ dev,  a staging and at least one production environment. The HPE Developer Community actually started with only a test/dev/staging on one side and a production on the other side.
+
+I﻿ will not focus here on the first steps. I leave it to you to figure out new subjects. I will talk a bit again about the infrastructure and especially the dedicated scripts and variables that you need to create to support the lifecycle of the workshop. As usual, there are two sides to the workshop's creation. What should be done on the backend and what needs to be done on the frontend (web portal and database server mainly)
+
+![](/img/wod-blogserie3-archi3.png "WOD Overview.")
+
+I﻿ will consider two scenarios going further. In the first one, I will create a simple workshop that does not require any infrastructure but the JupyterHub itself. Then, in a second phase, I will go through a more complex workshop creation process that will cover most of the possible cases I could think of.
+
+# S﻿imple workshop example:
+
+l﻿et's consider that I plan to create a new workshop on the Go language. It becomes more and more popular among the developer community I interact with and one of the the developer was kind enough to agree with working with me on creating this new workshop. After a first meeting, where I explainied to him the creation process, and the expectations, we quickly started to work together. We defined title, abstract,  notebooks' folder name, and student range. As far as infrastructure's requirements, a new kernel was needed. No additional scripts are requiered for this workshop.
```

Comment:
Make sure you capitalize the initial letter at the start of the first sentence - "Let's"

Instead of the word "consider", I think what you really mean is "imagine". So it would be, "Let's imagine that I plan to..."

Instead of "on the Go language", it should be "in the Go language." or just "in Go."

I would change the second sentence to read "Go has become more and more popular among..."

In the second line there, you have two "the" prior to "developer was kind enough". Since you're saying "one of the", it should be "one of the developers" (plural) or you could simply say "and one developer was kind enough"

Instead of "After a first meeting", I would change it to "After our first meeting".

In the third line, there's a typo in "explained" and it should be rephrased to "where I explained the creation process and expectations to him,"

You might want to modify the end of that sentence to "we were able to quickly start working together."

Add the word "the" just after "defined" (prior to "title, abstract..")

Place the word "the" prior to "infrastructure's"

In the last sentence, change "are requiered" to "were required" (changing one word and fixing the typo in required)